### PR TITLE
[003] - Onboarding updates 

### PR DESCRIPTION
**Pull request by ansonyuu**
-updated the text 
-updated the video to include toggle menu and thank you screen
-fixed timing issues from before with 3rd slide

### CODE REVIEW

**Comment by jackyzha0** [comment]
@ansonyuu did you test if the video still syncs with the text?

File: onboarding.html [hunk 1]
```diff
@@ -30,17 +30,17 @@
...
-        <p>on this page, you can add distracting websites or change how long you’re allowed into those website for. you can also get here by right-clicking the reflect icon and selecting 'show filter list.'</p>
+        <h2>on and off.</h2>
+        <p>to turn the extension off for longer breaks, press the icon in the upper right of your chrome menu to open the toggle menu.</p>
         <a id="page2button" class="lt-hover white_button shadow">next.</a>
       </div>
       <div id="page3" class="text-section">
-        <h2>on and off.</h2>
-        <p>to turn the extension off for longer breaks, press the icon in the upper right of your chrome menu.</p>
+        <h2>options.</h2>
+        <p>on this page, you can add distracting websites or change how long you’re allowed into those website for. you can also view your latest intents and set how many intents you want to display.</p>
```
Comment (jackyzha0) [inline onboarding.html:43]: maybe rephrase so that main purpose of options page is to customize your experience with reflect, then talk about what else it does

Comment (ansonyuu) [reply onboarding.html:43]: ooo i like this
